modules/kmm-tuning-the-module-resource.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * hardware_enablement/kmm-kernel-module-management.adoc
+
+:_content-type: PROCEDURE
+[id="kmm-tuning-the-module-resource_{context}"]
+= Tuning the Module resource
+
+.Procedure
+
+* Set `.spec.moduleLoader.container.modprobe.firmwarePath` in the `Module` custom resource (CR):
++
+[source,yaml]
+----
+apiVersion: kmm.sigs.x-k8s.io/v1beta1
+kind: Module
+metadata:
+  name: my-kmod
+spec:
+  moduleLoader:
+    container:
+      modprobe:
+        moduleName: my-kmod  # Required
+
+        firmwarePath: /firmware <1>
+
+  selector:
+    node-role.kubernetes.io/worker: "" <2>
+----
+<1> Optional: Copies `/firmware/*` into `/var/lib/firmware/` on the node.
+<2> Adds the kernel mappings.
```

Comment:
I would remove the <2> from there along with the selector field.
It is clearly specifying where firmwarePath needs to he added.

Also there is no `kernelMappings` in that example.

Reply:
Removed the callout and the `selector` field.